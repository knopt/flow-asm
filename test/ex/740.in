2
1
0.9874140535354705
2.325623295842851 -4.6188193481624324 

-4.610324896632331 
