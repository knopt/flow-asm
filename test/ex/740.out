-4.523029424001089 2.238220912499405 
